Airports/EGBS/Runway.txt

```diff
@@ -0,0 +1,2 @@
+08  26  083 263 N052.14.28.270 W002.53.14.310 N052.14.31.700 W002.52.30.590
+08G 26G 083 263 N052.14.30.129 W002.53.15.305 N052.14.33.402 W002.52.33.785
```

Comment:
Should we not just define all runways in the AIP as a general rule, otherwise we'll end up with a ridiculous number? Can't think of a reason why we'd need (unlicensed) grass strips selectable in the runway dialog

Reply:
Well, maybe I should've seen that one coming! Grass runways removed.